config.go

```diff
@@ -62,13 +66,31 @@ type ExecDWriter interface {
 	Write(value map[string]string) error
 }
 
+//go:generate mockery --name Logger --case=underscore
+
+// Logger is the interface implement by a type that wishes to write log messages generated by libcnb
+type Logger interface {
+	// Debug formats using the default formats for its operands
+	Debug(a ...interface{})
+
+	// Debugf formats according to a format specifier
+	Debugf(format string, a ...interface{})
+
+	// DebugWriter returns a writer where everything is written at this level
+	DebugWriter() io.Writer
```

Comment:
I can't find where this is used in the codebase. Do we need this in the interface?

Reply:
I don't think it's used in libcnb at the moment, so it's something we could potentially cut. I had mostly kept it as it was in the original log implementation.

The case where this tends to be useful is when running external commands. You can fetch the debug writer and pass that in as the Writer for output of the command. Looking at libcnb as a go implementation of the spec, it's probably unlikely we'd need this. There's nothing I can think of in the spec, at the moment, that would require us to run an external command.

I'll take that as one vote for removing it. I don't feel particularly strongly either way. Does anyone else feel strongly one way or another?

Reply:
Happy to get this removed as well.

Reply:
Cool, I'll pull it out.